### PR TITLE
feat(CC): add a new resource to manage bandwidth package

### PR DESCRIPTION
**Pull request by hmbig2**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [X] Tests added/passed.
* [X] Documentation updated.
* [X] Schema updated.

## Acceptance Steps Performed

```
Running tool: /usr/local/go/bin/go test -timeout 9000000000s -run ^TestAccBandwidthPackage_basic$ github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/cc -v

=== RUN   TestAccBandwidthPackage_basic
=== PAUSE TestAccBandwidthPackage_basic
=== CONT  TestAccBandwidthPackage_basic
--- PASS: TestAccBandwidthPackage_basic (24.51s)
PASS
ok  	github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/cc	24.668s
```

### CODE REVIEW

File: docs/resources/cc_bandwidth_package.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+   Valid value is **cloud_connection**.
+
+* `interflow_mode` - (Optional, String, ForceNew) Bandwidth package applicability.  
+  The options are as follows:
+    + **Area**: Geographic region.
+    + **Region**: Cloud region.
+
+  Changing this parameter will create a new resource.
+
+* `spec_code` - (Optional, String, ForceNew) Specification code of the bandwidth package.
```
Comment (ShiChangkuo) [inline docs/resources/cc_bandwidth_package.md]: is the spec_code useful? I didn't find it in console.

File: docs/resources/cc_bandwidth_package.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    + **3**: pay-per-use for the Chinese Mainland website.
+    + **4**: pay-per-use for the International website.
+    + **5**: 95th percentile bandwidth billing for the Chinese Mainland website.
+    + **6**: 95th percentile bandwidth billing for the International website.
+
+  -> This argument can only be modified to **5** and **6**.
+
+* `bandwidth` - (Required, Int) Bandwidth in the bandwidth package.  
+
+* `project_id` - (Required, String, ForceNew) Project ID.
```
Comment (ShiChangkuo) [inline docs/resources/cc_bandwidth_package.md]: the CC service is global, why should we input the project_id and what's the valid value of this parameter?

File: docs/resources/cc_bandwidth_package.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+## Example Usage
+
+```hcl
+variable "project_id" {}
+
+resource "huaweicloud_cc_bandwidth_package" "test" {
+  name           = "demo"
+  local_area_id  = "Chinese-Mainland"
+  remote_area_id = "Chinese-Mainland"
+  interflow_mode = "Area"
```
Comment (ShiChangkuo) [inline docs/resources/cc_bandwidth_package.md]: the local area and remove area are the same, why we set `interflow_mode` to **Area**?

File: docs/resources/cc_bandwidth_package.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+subcategory: "Cloud Connect (CC)"
+---
+
+# huaweicloud_cc_bandwidth_package
+
+Manages a bandwidth package resource of Cloud Connect within HuaweiCloud.  
+
+## Example Usage
+
+```hcl
```
Comment (ShiChangkuo) [inline docs/resources/cc_bandwidth_package.md:11]: it's better to add an example for cross area

File: docs/resources/cc_bandwidth_package.md (hunk 3 above)
Comment (hmbig2) [reply docs/resources/cc_bandwidth_package.md]: deleted

File: docs/resources/cc_bandwidth_package.md (hunk 1 above)
Comment (hmbig2) [reply docs/resources/cc_bandwidth_package.md]: deleted

File: docs/resources/cc_bandwidth_package.md (hunk 2 above)
Comment (hmbig2) [reply docs/resources/cc_bandwidth_package.md]: change to optional

File: docs/resources/cc_bandwidth_package.md (hunk 4 above)
Comment (hmbig2) [reply docs/resources/cc_bandwidth_package.md:11]: now, the cross area is the same as the example

File: huaweicloud/services/cc/resource_huaweicloud_cc_bandwidth_package.go [hunk 5]
```diff
@@ -0,0 +1,461 @@
...
+				Optional:    true,
+				Computed:    true,
+				Description: `Type of the resource that the bandwidth package is bound to.`,
+			},
+			"tags": {
+				Type:        schema.TypeMap,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				Optional:    true,
+				Computed:    true,
+				ForceNew:    true,
```
Comment (ShiChangkuo) [inline huaweicloud/services/cc/resource_huaweicloud_cc_bandwidth_package.go]: we can modify the tags through https://support.huaweicloud.com/api-cc/BatchCreateDeleteTags.html

Comment (hmbig2) [reply huaweicloud/services/cc/resource_huaweicloud_cc_bandwidth_package.go]: this API is invalid.

**Pull request merged**